action.yml

```
@@ -55,6 +61,17 @@ runs:
           mesa-utils \
           x11-utils
 
+    - name: Determine OpenGL version to install on Windows
+      if: runner.os == 'Windows'
+      shell: bash
+      run: |
+        if [ "${{ inputs.mesa3d-release }}" == "latest" ]; then
+          export MESA3D_VERSION=$(curl -s https://api.github.com/repos/pal1000/mesa-dist-win/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
+        else
+          export MESA3D_VERSION=${{ inputs.mesa3d-release }}
+        fi
+        echo "MESA3D_VERSION=${MESA3D_VERSION}" >> $GITHUB_ENV
```

I like to do this because then I get to see what's going in there, can be really nice for debugging down the line
```suggestion
        echo "MESA3D_VERSION=${MESA3D_VERSION}" | tee -a $GITHUB_ENV
```